packages/react-shopify-app-route-propagator/src/test/react-shopify-app-route-propagator.test.tsx

```diff
@@ -1,5 +1,5 @@
 import * as React from 'react';
-import {mount} from 'enzyme';
+import {mount} from '@shopify/react-testing';
```

Comment:
can we do a beta release of `@shopify/react-testing` and iterate on it?

Reply:
I'll get a release out today 👍